cmd/gapir/cc/main.cpp

```diff
@@ -57,6 +57,8 @@
 using namespace core;
 using namespace gapir;
 
+const size_t RAM_CACHE_SIZE = 256ul * 1024ul * 1024ul;
```

Comment:
I would strongly prefer we don't reduce our default cache sizes. This can severely affect our performance, especially as we build out our profiling. If we can stick with one or 2 long stalls, that will make performance easier to gauge than many shorter stalls.

Reply:
@AWoloszyn please consult https://docs.google.com/spreadsheets/d/1RgAgwsQmUjf-gs_P5RVg7Qmsh2NDhmlAImlZI4dE8zg/edit#gid=0

The performance of this new cache system is far greater than the old system, even with greatly reduced cache sizes. Because it is not working within a single alloc like the old system though, LMK can be more aggressive if we do not keep the memory pressure low. That sheet shows cache hit performance of >= 99.95% even with cache sizes as small as 128mb.

Reply:
@hevrard If a texture is too large to fit in the cache, loadCache() will return false and the resource will be fetched as a one-off uncached transaction. I think this is a good thing as we don't want to blow our cache with gigantic single resources. Empirically in my tests, the reused resources are the small ones. Larger resources tend to be one shot. And it's actually the ADB send-receive overheads that slow us down, not the actual data transfer, so we're always better using N bytes to cache 1000 small resources than a single large one.